src/reuse/helper/errorHandler.ts

```diff
@@ -21,19 +34,19 @@ export default class ErrorHandler implements IErrorHandler {
     this.logStackTrace = logStackTrace;
   }
 
-  public logException(errorObject: Error): Promise<never> {
+  public logException(errorObject: Error): never {
     if (errorObject) {
       let functionName = this._retrieveFunctionNameFromStack(errorObject);
 
-      const stackTrace = this.logStackTrace === true ? this._getFormattedStackTrace(errorObject) : "";
+      const stackTrace = this.logStackTrace === true ? "displayStack" : undefined;
```

Comment:
what is the "displayStack" string used for? As indicator that it should display the stack or not?

Reply:
@DIFSRIP I have changed the displaystack to boolean value